api/websocket/server/server.go

```diff
@@ -487,19 +487,8 @@ func (ws *WsServer) Broadcast(data []byte) error {
 }
 
 func (ws *WsServer) initTlsListen() (net.Listener, error) {
-
-	CertPath := config.Parameters.HttpWssCert
-	KeyPath := config.Parameters.HttpWssKey
-
-	// load cert
-	cert, err := tls.LoadX509KeyPair(CertPath, KeyPath)
-	if err != nil {
-		log.Error("load keys fail", err)
-		return nil, err
-	}
-
 	tlsConfig := &tls.Config{
-		Certificates: []tls.Certificate{cert},
+		GetCertificate: api.ReturnCert,
```

Comment:
Previously wss cert (HttpWssCert) might be different from https json cert (HttpsJsonCert), but now HttpWssCert is no longer being used